src/Groups/pcgroup.jl

```diff
@@ -923,3 +923,57 @@ function collector(::Type{T}, G::PcGroup) where T <: IntegerUnion
 end
 
 collector(G::PcGroup) = collector(ZZRingElem, G)
+
+# GAP wrappers for group encoding / decoding
+
+"""
+    code_pcgroup(G::PcGroup)
+
+Return a `ZZRingElem` representing the polycyclic group `G`, using the same
+encoding as GAP's `CodePcGroup` and Magma's `SmallGroupEncoding`.
+Currently only defined for `PcGroup`, not `SubPcGroup`.
+
+# Examples
+```jldoctest
+julia> G = small_group(12, 2)
+Pc group of order 12
+
+julia> code = code_pcgroup(G)
+266
+
+julia> H = pcgroup_code(code, order(G))
+Pc group of order 12
+
+julia> code_pcgroup(G) == code_pcgroup(H)
+true
+```
+"""
+function code_pcgroup(G::PcGroup)
+  return ZZ(GAP.Globals.CodePcGroup(GapObj(G))::GapInt)
+end
+
+"""
+   pcgroup_code(code::IntegerUnion, order::IntegerUnion)
+
+Given an integer `code` and an integer `order`, return the polycyclic group it encodes.
+Both `code` and `order` can be of type `Int`, `BigInt`, or `ZZRingElem`. Internally, these are converted to `GapInt` before calling the GAP function `PcGroupCode`.
```

Comment:
That's an implementation details that does not need to be mentioned here, and in fact should not be mentioned (I am afradi the non-technical user will be confused by it -- asking questions like "but what about `UInt`, does that mean it is not supported?")
```suggestion
Both `code` and `order` can be of type `Int`, `BigInt`, or `ZZRingElem`. Internally, these are converted to `GapInt` before calling the GAP function `PcGroupCode`.
```